tests/basic_fuzzer.cpp

```diff
@@ -118,11 +119,11 @@ size_t roller_fuzz(size_t N) {
 }
 
 int main() {
-#if ADA_IS_BIG_ENDIAN
-  std::cout << "You have big-endian system." << std::endl;
-#else
-  std::cout << "You have litte-endian system." << std::endl;
-#endif
+  if (std::endian::native == std::endian::big) {
```

Comment:
Not that it matters/changes anything

```suggestion
  if constexpr (std::endian::native == std::endian::big) {
```